hws/mlx_hw_steering_parser.py

```diff
@@ -19,7 +19,7 @@
 from src.dr_rule import *
 from src.dr_hw_resources import *
 from src.dr_ste import *
-from src.dr_db import _config_args, _fw_ste_indexes_arr, _stc_indexes_arr
```

Comment:
Again, no way to put comment in commit message - these questions would have been better suited there.
Anyway: how will it work? All the context's are printed into the same file? Any implication on other parameters for dump tool?
Please document this both in commit message and (more importantly) in the readme.

Reply:
Done